metadata/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "subxt-metadata"
+version = "0.20.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+autotests = false
+
+license = "GPL-3.0"
+repository = "https://github.com/paritytech/subxt"
+documentation = "https://docs.rs/subxt"
+homepage = "https://www.parity.io/"
+description = "Command line utilities for checking metadata compatibility between nodes."
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = ["derive", "full"] }
+frame-metadata = "15.0.0"
+scale-info = "2.0.0"
+sp-core = { version = "6.0.0"  }
+
+[dev-dependencies]
+bitvec = { version = "1.0.0", default-features = false, features = ["alloc"] }
+criterion = "0.3"
+scale-info = { version = "2.0.0", features = ["bit-vec"] }
+test-runtime = { path = "../test-runtime" }
```

Comment:
Unused?

Reply:
`test-runtime` is used currently to load metadata for benches.

Reply:
Ah yes I see